<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Map</class>
 <widget class="QWidget" name="Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <widget class="QMenuBar" name="menuBar"/>
   </item>
   <item>
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="PutMarker">
         <property name="text">
          <string>Place Waypoint</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearMarker">
         <property name="text">
          <string>Clear Waypoints</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="DrawShape">
         <property name="text">
          <string>Input Boundary Coordinates</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBound">
         <property name="text">
          <string>Cancel Boundary Coordinates</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeBoundary">
         <property name="text">
          <string>Remove Boundary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DrawNoFly">
         <property name="text">
          <string>Input No-Fly Zone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelNoFly">
         <property name="text">
          <string>Cancel No-Fly Zone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeNoFly">
         <property name="text">
          <string>Remove No-Fly Zone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="finishRemove">
         <property name="text">
          <string>Finish</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="inputLatLabel">
               <property name="text">
                <string>Input Latitude:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="latBound"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="inputLongLabel">
               <property name="text">
                <string>Input Longitude:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="longBound"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="confirmBound">
               <property name="text">
                <string>Finish</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="addBound">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="undoBound">
             <property name="text">
              <string>Undo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="copyBound">
             <property name="text">
              <string>Copy from Current Coordinates</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="inputLatLabel_2">
               <property name="text">
                <string>Input Latitude:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="latNoFly"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="inputLongLabel_2">
               <property name="text">
                <string>Input Longitude:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lngNoFly"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>17</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="confirmNoFly">
               <property name="text">
                <string>Finish</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="addNoFly">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="undoNoFly">
             <property name="text">
              <string>Undo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="copyNoFly">
             <property name="text">
              <string>Copy from Current Coordinates</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="boundCoordinates"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current Latitude:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Current Longitude:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lat"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lng"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
